Apps/Common/src/Services/CommunicationService.cs

```diff
@@ -142,7 +142,8 @@ public void ProcessChange(BannerChangeEvent changeEvent)
                     {
                         this.logger.LogInformation("{Action} ChangeEvent for Communication {Id} found in Cache", changeEvent.Action, communication.Id);
                         this.RemoveCommunicationFromCache(communication.CommunicationTypeCode);
-                        if (changeEvent.Action is Insert or Update)
```

Comment:
I think this may cause a problem with hitting the DB when no result is cached... we probably need to create an empty cache item

Reply:
The result is the same as when doing a Delete. Not sure what DB problem this would cause.